tests/functional/z_cluster/cluster_expansion/test_add_capacity.py

```
@@ -79,15 +93,24 @@ def add_capacity_test(ui_flag=False):
+        f"restarted={restarted_osds}"
+    )
     assert (
         len(restarted_osds) == 0
     ), f"The following OSD pods were restarted (deleted) post add capacity: {restarted_osds}"
 
+    logger.test_step("Wait for new OSD pods to reach Running state")
     pod = OCP(kind=constants.POD, namespace=config.ENV_DATA["cluster_namespace"])
     if is_flexible_scaling_enabled():
         replica_count = 1
     else:
         replica_count = 3
```

Ruff (0.16.0)

[warning] Use ternary operator `replica_count = 1 if is_flexible_scaling_enabled() else 3` instead of `if`-`else`-block

Replace `if`-`else`-block with `replica_count = 1 if is_flexible_scaling_enabled() else 3`

(SIM108)